pyproject.toml

```diff
@@ -1,17 +1,39 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools>=64",
     "wheel",
-    "setuptools_scm[toml]>=3.4",
+    "setuptools_scm[toml]>=8.0",
```

Comment:
I don't think that we need any of these build dependencies anymore!

Reply:
(I specifically mean `setuptools`, `wheel`, and `setuptools_scm`.)

Reply:
Oh wow, that's pretty cool. I didn't understand how this could work without specifying at least `setuptools_scm`, and after reading through the source code I now understand that build backends can dynamically tell pip to install extra dependencies. Cool!